config/routes.rb

```diff
@@ -5,4 +5,5 @@
   resources :restaurants, only: [:new, :create]
   root controller: :landing, action: :index
   resources :menus, only: [:new, :create]
+  resources :dishes, only: [:new, :create]
```

Comment:
This route should have been created when needed, meaning that there should be an acceptance test that tries toi access it before you actually create it.